pkgs/applications/video/mov-cli/default.nix

```
@@ -0,0 +1,29 @@
+{ lib
+, python310
+, fetchFromGitHub
+}:
+
+python310.pkgs.buildPythonPackage rec {
```

```suggestion
python3.pkgs.buildPythonPackage rec {
```
that is the default already